pkg/lightning/config/config.go

```diff
@@ -82,7 +82,7 @@ const (
 	// defaultMetaSchemaName is the default database name used to store lightning metadata
 	defaultMetaSchemaName     = "lightning_metadata"
 	defaultTaskInfoSchemaName = "lightning_task_info"
-	defaultMaxRecordRows      = 100
+	DefaultThreshold          = 10000
```

Comment:
name it DefaultRecordDuplicateThreshold or something, to make its usage more obvious